test/contract/logs/pact.log

```diff
@@ -0,0 +1,58 @@
+I, [2020-04-19T14:23:48.892115 #279]  INFO -- : Registered expected interaction GET /schedule
+D, [2020-04-19T14:23:48.894272 #279] DEBUG -- : {
+  "description": "get all schedules",
+  "providerState": "schedules exist",
+  "request": {
+    "method": "GET",
+    "path": "/schedule"
+  },
+  "response": {
+    "status": 200,
+    "headers": {
+      "Content-Type": "application/json; charset=utf-8"
+    },
+    "body": {
+      "json_class": "Pact::ArrayLike",
+      "contents": {
+        "claimId": {
+          "json_class": "Pact::SomethingLike",
+          "contents": "MINE123"
+        }
+      },
+      "min": 1
+    }
+  },
+  "metadata": null
+}
+I, [2020-04-19T14:23:48.910350 #279]  INFO -- : Received request GET /schedule
+D, [2020-04-19T14:23:48.911385 #279] DEBUG -- : {
+  "path": "/schedule",
+  "query": "",
+  "method": "get",
+  "headers": {
+    "Authorization": "Bearer token",
+    "Host": "127.0.0.1:1234",
+    "Connection": "close",
+    "Version": "HTTP/1.1"
+  }
+}
+I, [2020-04-19T14:23:48.914712 #279]  INFO -- : Found matching response for GET /schedule
+D, [2020-04-19T14:23:48.916015 #279] DEBUG -- : {
+  "status": 200,
+  "headers": {
+    "Content-Type": "application/json; charset=utf-8"
+  },
+  "body": {
+    "json_class": "Pact::ArrayLike",
+    "contents": {
+      "claimId": {
+        "json_class": "Pact::SomethingLike",
+        "contents": "MINE123"
+      }
+    },
+    "min": 1
+  }
+}
+I, [2020-04-19T14:23:48.927773 #279]  INFO -- : Verifying - interactions matched
+I, [2020-04-19T14:23:48.935182 #279]  INFO -- : Cleared interactions
+I, [2020-04-19T14:23:48.943478 #279]  INFO -- : Writing pact for ffc-demo-payment-service to /home/node/test/contract/pacts/ffc-demo-payment-web-ffc-demo-payment-service.json
```

Comment:
In the walkthrough, the logs and pacts directories are added to the .gitignore file